src/testing/euler_split_inviscid_taylor_green_vortex.h

```diff
@@ -55,6 +55,7 @@ class EulerTaylorGreen : public TestsBase
     /** Overintegration of kinetic energy.
      */
  double compute_kinetic_energy(std::shared_ptr < DGBase<dim, double> > &dg, unsigned int poly_degree) const;
+ double compute_MK_energy(std::shared_ptr < DGBase<dim, double> > &dg, unsigned int poly_degree) const;
```

Comment:
what is MK? could you rename this without the abbreviations?

Reply:
fixed